build-support/bin/common.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Note this file is adapted from https://github.com/pantsbuild/setup/blob/gh-pages/build-support/common.py. I removed all the parts that aren't relevant to Pants repo.

Currently we are not using the `travis_section()` and `elapsed_time()` functions, but I still included them to make it easier for future scripts to use and because they are already battle-tested. I can remove if need be, but I fear others who might write Python scripts wouldn't think to check the setup repo to see that we already have the code written.